src/ContainerDetails.jsx

```diff
@@ -27,44 +27,50 @@ const render_container_published_ports = (ports) => {
 
 // container.Created is an ISO datetime string, like 2021-07-11T14:17:00.350804119+02:00
 const ContainerDetails = ({ container, containerDetail }) => (
-    <DescriptionList isHorizontal className='container-details'>
-        <DescriptionListGroup>
-            <DescriptionListTerm>{_("ID")}</DescriptionListTerm>
-            <DescriptionListDescription>{container.Id}</DescriptionListDescription>
-        </DescriptionListGroup>
-        <DescriptionListGroup>
-            <DescriptionListTerm>{_("Created")}</DescriptionListTerm>
-            <DescriptionListDescription>{utils.localize_time(Date.parse(container.Created) / 1000)}</DescriptionListDescription>
-        </DescriptionListGroup>
-        <DescriptionListGroup>
-            <DescriptionListTerm>{_("Image")}</DescriptionListTerm>
-            <DescriptionListDescription>{container.Image}</DescriptionListDescription>
-        </DescriptionListGroup>
-        <DescriptionListGroup>
-            <DescriptionListTerm>{_("Command")}</DescriptionListTerm>
-            <DescriptionListDescription>{container.Command ? utils.quote_cmdline(container.Command) : ""}</DescriptionListDescription>
-        </DescriptionListGroup>
-        <DescriptionListGroup>
-            <DescriptionListTerm>{_("State")}</DescriptionListTerm>
-            <DescriptionListDescription>{render_container_state(container)}</DescriptionListDescription>
-        </DescriptionListGroup>
-        <DescriptionListGroup>
-            <DescriptionListTerm>{_("Ports")}</DescriptionListTerm>
-            <DescriptionListDescription>{render_container_published_ports(container.Ports)}</DescriptionListDescription>
-        </DescriptionListGroup>
-        <DescriptionListGroup>
-            <DescriptionListTerm>{_("IP address")}</DescriptionListTerm>
-            <DescriptionListDescription>{containerDetail ? containerDetail.NetworkSettings.IPAddress : ""}</DescriptionListDescription>
-        </DescriptionListGroup>
-        <DescriptionListGroup>
-            <DescriptionListTerm>{_("Gateway")}</DescriptionListTerm>
-            <DescriptionListDescription>{containerDetail ? containerDetail.NetworkSettings.Gateway : ""}</DescriptionListDescription>
-        </DescriptionListGroup>
-        <DescriptionListGroup>
-            <DescriptionListTerm>{_("MAC address")}</DescriptionListTerm>
-            <DescriptionListDescription>{containerDetail ? containerDetail.NetworkSettings.MacAddress : ""}</DescriptionListDescription>
-        </DescriptionListGroup>
-    </DescriptionList>
+    <Flex spaceItems={{ modifier: 'spaceItemsXl' }}>
```

Comment:
I must say it seems a bit odd to not fill up the card but have items in columns:
![Screenshot from 2021-09-08 09-33-10huh](https://user-images.githubusercontent.com/12330670/132466632-772f6c1d-6e22-48c8-b594-07c736099540.png)

Shouldn;t we put those items on one line if there is space? I know this follows up the design -  @garrett

Reply:
That doesn't follow the design though... the design has it fill the space.

Here's the design:
![podman-redesign-2021](https://user-images.githubusercontent.com/10246/132471442-f7c65365-2096-4be4-adc9-af44bf9fba46.png)

Here's how it would wrap when space gets limited:

![podman-redesign-2021-details-wrapping](https://user-images.githubusercontent.com/10246/132471509-b1d4824d-4430-4d6a-a721-35f93eb3ff63.png)

The idea is that there are groups of similar information and both the contents of the group and the group itself wraps when necessary.

Reply:
@garrett the reason I did not follow the design here is that the design looks like it uses a justify-content: space-between

This works nicely when all info are present - but when the whole networking info is missing ( which is very often the case ) then there is a ton of unused in-between space.

Reply:
It can use nested PF4 flex elements, each with gaps. I agree about space-between when info isn't there.

Anyway, it's not a big deal, and we can iterate to optimize it later, in another PR.

Reply:
I am not sure how it should behave differently than what we have now. I will let it to you @garrett to work on this when you have time.